packages/docusaurus-1.x/lib/core/Doc.js

```diff
@@ -238,7 +238,9 @@ class Doc extends React.Component {
         <header className="postHeader">
           {editLink}
           {!this.props.hideTitle && (
-            <h1 className="postHeaderTitle">{this.props.title}</h1>
+            <h1 id="__docusaurus" className="postHeaderTitle">
```

Comment:
This need to be reverted.

Reply:
@lex111 - Thanks for reviewing it. Could you please give an explanation on why this needs to be reverted?

The title of the page is "part of the page content". When you integrate with Algolia DocSearch, Algolia generate a permalink if the search query matches the page title. That's why they recommend having all headings to have an `id`. Please see the related issue for a detailed discussion.

Reply:
@parvezakkas I see, thanks for the clarification!